api-tests/dev_apis/internal_trusted_storage/test_s010/test_its_data.h

```diff
@@ -29,7 +29,7 @@ typedef struct {
 
 static const test_data s010_data[] = {
 {
-  VAL_ITS_SET, PSA_ITS_SUCCESS /* Create with UID value zero should fail */
+  VAL_ITS_SET, PSA_ITS_SUCCESS /* Create with UID value zero should pass */
```

Comment:
This should be fail only, since as per spec we cannot use UID value as 0
In test we are checking NOT_EQUAL condition.